Fix dotnet store options

eerhardt opened the pull request:
- remove configuration
- remove preserve-working-dir
- add skip-symbols

Fix #6488
Fix #6489
Fix #6490

Also fixes #1 of https://github.com/dotnet/sdk/issues/1126#issuecomment-299173452.  And along with https://github.com/dotnet/sdk/pull/1231 should complete the feature.

@gkhanna79 @ramarag 

/cc @bleroy @dotnet/dotnet-cli

livarcocc commented:
@eerhardt We are doing the ask mode template now for approval. You can see an example here: https://github.com/dotnet/cli/pull/6627. After filling up the template, tag matt for approval.

eerhardt commented:
**Customer scenario**

When first implementing `dotnet store`, we added 2 command line options that are confusing.  `configuration` doesn't do anything for the command.  And `preserve-working-dir` is an advanced/debugging scenario.

Removing those 2 options.

Also adding a new option to skip generating profiling symbols, as spec'd by the PM.

**Bugs this fixes:** 

Fix #6488
Fix #6489
Fix #6490

Also fixes #1 of dotnet/sdk#1126 (comment). And along with dotnet/sdk#1231 should complete the feature.

**Workarounds, if any**
N/A

**Risk**

Low. The removed options aren't used by users, and we just added the whole command in 2.0.

**Performance impact**
None.

**Is this a regression from a previous update?**
No

**Root cause analysis:**
Planned Feature

**How was the bug found?**
Planned Feature


@MattGertz for approval
